crates/pera-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2561,15 +2568,15 @@ impl AuthorityPerEpochStore {
         authority_metrics: &Arc<AuthorityMetrics>,
     ) -> PeraResult<Vec<VerifiedExecutableTransaction>> {
         // Split transactions into different types for processing.
-        let verified_transactions: Vec<_> = transactions
-            .into_iter()
-            .filter_map(|transaction| {
-                self.verify_consensus_transaction(
-                    transaction,
-                    &authority_metrics.skipped_consensus_txns,
-                )
-            })
-            .collect();
+        let mut verified_transactions: Vec<VerifiedSequencedConsensusTransaction> = vec![];
```

Comment:
Replaced the filter_map with this less-elegant alternative since async closures are not yet supported in Rust